resources/views/dashboard/training/new_ticket.blade.php

```diff
@@ -80,6 +80,24 @@
                 </div>
             </div>
         </div>
+        <div class="row">
+            <div class="col-sm-3">
+                <div class="form-group">
+                    <label for="score" class="form-label">Score (Visible to Controller)</label>
```

Comment:
For UX, we need to provide some context behind the score (ex. 5 is best, 1 is worst)